app/models/loan_request.rb

```diff
@@ -0,0 +1,16 @@
+class LoanRequest < ActiveRecord::Base
+  validates :title, :description, :amount,
+    :requested_by_date, :repayment_begin_date,
+    :repayment_rate, presence: true
+  has_attached_file :picture,
+                    styles: { large: "960x960",
+                              medium: "300x300>",
+                              thumb: "100x100>" },
+                              default_url: "public/seed_photos/:style/missing.png"
```

Comment:
Line is too long. [82/80]<br>Align the elements of a hash literal if they span more than one line.